src/cpu/pred/BranchPredictor.py

```diff
@@ -93,6 +103,13 @@ class SimpleBTB(BranchTargetBuffer):
     instShiftAmt = Param.Unsigned(
```

Comment:
This is entry_size in the indexing policy, so this parameter should be removed and instantiated only there, or it should be used like you do for tagBits (i.e., entry_size=Self.instShiftAmt)